openupgrade_scripts/apriori.py

```diff
@@ -34,6 +34,7 @@
     "viin_product_categ_mail_thread_stock_account": "viin_mail_thread_stock_account",
     "viin_webp": "viin_web_editor",
     "viin_user_assignment_log_project": "test_viin_user_assignment_log_project",
+    "to_stock_production_lot_partner_infor": "viin_stock_lot_partner_infor",
```

Comment:
Please preserve the order for readability, move this after `to_stock_picking_backdate`i guest